### PR TITLE
[Behats][Api] handling last picked up cart

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.8
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no?
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Shop/CartContext.php [hunk 1]
```diff
@@ -143,6 +143,22 @@ public function iRemoveProductFromTheCart(ProductInterface $product, string $tok
...
+    /**
+     * @When I pick up my cart (again)
+     */
+    public function iPickUpMyCart(): void
+    {
+        $this->pickupCart();
+    }
+
+    /**
+     * @Given /^I check details of my (cart)$/
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Api/Shop/CartContext.php:155]:
```suggestion
     * @When /^I check details of my (cart)$/
```

File: src/Sylius/Behat/Context/Api/Shop/CartContext.php [hunk 2]
```diff
@@ -323,6 +339,26 @@ public function myCartShouldHaveItems(int $quantity, ProductInterface $product):
...
     }
 
+    /**
+     * @Then /^I should have empty (cart)$/
+     */
+    public function iShouldHaveEmptyCart(string $tokenValue): void
+    {
+        $items = $this->responseChecker->getValue($this->cartsClient->show($tokenValue), 'items');
+
+        Assert::same(count($items), 0, 'There should be a empty cart');
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Api/Shop/CartContext.php:349]:
```suggestion
        Assert::same(count($items), 0, 'There should be an empty cart');
```

File: features/cart/shopping_cart/maintaining_last_picked_up_cart.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Feature: Maintaining a last picked up cart
+    In order to manage proper cart
+    As a Visitor
+    I want to be able to always maintain last picked up cart
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @api
+    Scenario: having access to a last picked up cart
```
Comment (GSadee) [inline features/cart/shopping_cart/maintaining_last_picked_up_cart.feature:11]:
```suggestion
    Scenario: Having access to a last picked up cart
```

**Comment by AdamKasp** [comment]
replaced by #11895

**Pull request closed without merging**